lib/Swagger/src/ContentEntity/Api/EntityMetadataApi.php

```diff
@@ -321,6 +326,11 @@ public function deleteWithHttpInfo($entity_id, $meta_data_name)
         } elseif (count($formParams) > 0) {
             $httpBody = $formParams; // for HTTP post (form)
         }
+        // this endpoint requires API key authentication
+        $apiKey = $this->apiClient->getApiKeyWithPrefix('AUTH-SECRET');
+        if (strlen($apiKey) !== 0) {
+            $headerParams['AUTH-SECRET'] = $apiKey;
+        }
         // make the API Call
         try {
             list($response, $statusCode, $httpHeader) = $this->apiClient->callApi(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$response" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)